### ISSUE TITLE
How to set the persistency profile to None

### ISSUE THREAD

**Issue by wheestermans31** [F5Networks/k8s-bigip-ctlr#2385]
Now we have the following policy crd 

...
spec:
  iRules:
    insecure: /Common/Proxy_Protocol_iRule
    priority: high
    secure: /Common/Proxy_Protocol_iRule
  profiles:
    persistenceProfile: source-address
...

How do I set the profile to None, using no profile at all? 
I only seems to have the following values allowed:

2022/05/02 11:18:35 [ERROR] [AS3] Raw response from Big-IP: map[code:422 declarationFullId: errors:[/KuberNetesNGINX/Shared/nginx_virtual_server_http_80/persistenceMethods/0: should be equal to one of the allowed values ["cookie","destination-address","msrdp","source-address","tls-session-id"]] message:declaration is invalid]

**Comment by mdditt2000** [comment]
this is resolved in CIS 2.8. Please watch the [YouTube video](https://www.youtube.com/watch?v=brlNGGJ0sw0) and [github doc](https://github.com/mdditt2000/kubernetes-1-19/tree/master/cis%202.8/crd-enhancements#readme)

Change your configuration to 

```
apiVersion: cis.f5.com/v1
kind: Policy
metadata:
  labels:
    f5cr: "true"
  name: policy-persistence
  namespace: default
spec:
  profiles:
    http: /Common/Custom_HTTP
    tcp: /Common/Custom_TCP
    persistenceProfile: none
```

**Comment by wheestermans31** [comment]
The value none did it. So we can close

But before I tried that and it gave me this error, but for some reason I'm not able to reproduce.

2022/05/02 11:18:35 [ERROR] [AS3] Raw response from Big-IP: map[code:422 declarationFullId: errors:[/KuberNetesNGINX/Shared/nginx_virtual_server_http_80/persistenceMethods/0: should be equal to one of the allowed values ["cookie","destination-address","msrdp","source-address","tls-session-id"]] message:declaration is invalid]
 
Anyway, it is working now, so we can close.
